data-access-layer/datastore/database.go

```diff
@@ -55,7 +55,7 @@ func (a DbRoleSlice) Len() int {
 	return len(a)
 }
 
-func (dbRole DbRole) IsTenantDbRole() bool {
+func (dbRole DbRole) isDbRoleTenantSpecific() bool {
```

Comment:
IsDbRoleTenantScoped()

Reply:
Renamed the method. Also renamed the booleans that take the value returned by this method to `IsDbRoleTenantScoped`